jb/research.md

```diff
@@ -70,51 +96,115 @@ Table 3: Evidence from Sub-Saharan Africa: Cash transfers reduce child material
+*Note: this can have more causal language than "associated" right?*
 
-On the intensive margin, there is some weak evidence that school attendance increases on receipt of the transfer (e.g. 13.4 pp\*\*\* for the CCT arm of the Burkino Faso NCTPP null result for the UCT arm and 13 pp\*\* Malawi SCTP {cite}`BurkinoFaso_School` {cite}`Malawi_SCTP_final`) though a clear majority of null results is reflected in the the World Bank meta-analysis finding no significant impact across the programs {cite}`World_Bank_meta`. Similarly mixed results are reported by these programs for grade attainment. 
+There is some evidence that cash transfers increase the probability that a child is enrolled in school.
+For example, the Malawi SCTP found that receipt of the transfer was associated with an 11 pp\*\*\* increase in enrollment from baseline and the Zambia MCTG an 8 pp\* increase {cite}`Malawi_SCTP_final` {cite}`Zambia_MCTG_final`.
+There is also evidence from the Burkino Faso NCTPP that, while enrollment increases for transfers conditional on child attendance (17.9 pp\*\*\*), there is no effect for unconditional transfers {cite}`BurkinoFaso_School`.
 
-Malawi's ZCTP also found improvements in varied scores of educational attainment and cognition for its conditional transfer program (which required school attendance), but not the unconditional transfer program (on average, roughly 0.15\*\*\* sd improvements) {cite}`MZCTP`. This, combined with evidence from the Burkino Faso NCTPP may indicate that if educational outcomes are the primary purpose of the cash transfer intervention, then conditioning the transfer on school attendance may be a more cost-efficient intervention. There is, however, evidence that these educational effects are once transfers stop {cite}`ZCTP_medium`. 
+*Note: are any of the below conditional on child attendance? If any other than NCTPP were, it'd be good to break out the UCTs into a single paragraph.*
 
+However, the evidence isn't conclusive.
+The Kenya CT-OVC and Ghana LEAP only found significant enrollment effects on children age 13 to 17 {cite}`Ghana_final` {cite}`Kenya_CT_OVC`.
+The Zimbabwe HSCT found no enrollment effects, though the authors indicate that this could be due to loss of income-based scholarships {cite}`Zimbabwe_HSCT_final`.
+Moreover, evidence from the Zambia CGP indicates that enrollment improvements did not persist: a 10 pp\*\* increase at 30 months declined to a non-significant 6 pp at 36 months.
```

Comment:
Needs inclusion of citation.
{cite}`Zambia_CGP_final`

Reply:
What would you think about putting the citations in the table once, and then referring to them by the program name (as is currently done)? They're getting referenced up to 10 times each, and it's looking a bit cluttered.

Reply:
Per our conversation, let's do this and then add a footnote describing which studies reported which outcome metrics if any programs have multiple studies.